.github/workflows/test_python.yml

```diff
@@ -38,8 +38,23 @@ jobs:
   Docker:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@master
+    - uses: actions/checkout@v2
     - name: Docker build
       run: docker build --tag=kivy/buildozer .
     - name: Docker run
       run: docker run kivy/buildozer --version
+
+  Integration:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Setup python
+      uses: actions/setup-python@v1
```

Comment:
**nitpick:** we could use `actions/setup-python@v2`

Reply:
Addressed :+1: